obp-api/src/main/scala/code/api/UKOpenBanking/v3_1_0/UtilForUKV310.scala

```diff
@@ -0,0 +1,61 @@
+package code.api.UKOpenBanking.v3_1_0
+
+import code.api.util.APIUtil.{canGrantAccessToViewCommon, canRevokeAccessToViewCommon}
+import code.api.util.ErrorMessages.UserNoOwnerView
+import code.views.Views
+import com.openbankproject.commons.model.{User, ViewIdBankIdAccountId}
+import net.liftweb.common.{Empty, Failure, Full}
+
+import scala.collection.immutable.List
+
+object UtilForUKV310 {
+  def grantAccessToViews(user: User, views: List[ViewIdBankIdAccountId]): Full[Boolean] = {
```

Comment:
Hi @constantine2nd 
There is another method has the same name `code.api.util.Consent.grantAccessToViews`, please check whether duplicated or should we merge to a single one?